### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by byiringiroscar**
#### What does this PR do ❓ 

This PR is raised to implement the following features 🆗 :

- [X] Page title 
- [X] Image text alternatives  (<img alt="alt text:>)
- [X] Text Headings
- [X]  color contrast 
- [X]  Resize text 
- [X] moving content 
- [X]  multimedia 
- [X] basic structure of the pages 

### How should it be tested ⬇️ ⬇️ ⬇️ 

clone this repository
     
```
git clone https://github.com/byiringiroscar/portfolio_desktop_microverse.github.io/
```
Navigate to folder
```
cd portfolio_desktop_microverse.github.io
```
Switch to this branch disability-accessibility
```
git switch disability-accessibility
```

Start project 😄 

#### Run index.html

Thanks for reviewing my PR 😄

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,19 +18,19 @@
...
             </nav>
         </header>
         <div class="main-body">
             <div class="social-icon">
                 <ul>
-                    <li><a href="https://github.com/byiringiroscar"><img src="./images/github.png"></a></li>
-                    <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin.png"></a></li>
+                    <li><a href="https://github.com/byiringiroscar"><img src="./images/github.png" alt="github"></a></li>
+                    <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin.png" alt="linkedin"></a></li>
                     <li><a href="#"><img src="./images/hand.png"></a></li>
```
Comment (mckent05) [inline index.html]: - The image on line 31 is missing an `alt` text, please fix this so it helps to improve the accessibility of your site 🙏

File: index.html [hunk 2]
```diff
@@ -238,13 +238,13 @@ <h3>I'm always interested in hearing about new projects, so if you'd like to cha
         </div>
         <div class="contact-footer">
             <ul>
-                <li><a href="https://github.com/byiringiroscar"><img src="./images/github1.png" alt=""></a></li>
-                <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin1.png" alt=""></a></li>
+                <li><a href="https://github.com/byiringiroscar"><img src="./images/github1.png" alt="github"></a></li>
+                <li><a href="https://www.linkedin.com/in/oscar-byiringiro-9baa8313a/"><img src="./images/linkedin1.png" alt="linkedin"></a></li>
                 <li><a href="#"><img src="./images/hand1.png" alt=""></a></li>
```
Comment (mckent05) [inline index.html]: - The image on line 243 is missing an `alt` text, please fix this to help improve the accessibility of your site 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @byiringiroscar, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
- According to the project requirements:
`In the result section of the report the student should include a statement pertaining to every point checked:
  1  If everything was ok, they make a statement about that (e.g. "Page titles - no issues found").
  2 If issues were found, they should link to the corresponding line of code in the Pull Request changes`
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**